examples/p4est_3d_dgsem/elixir_euler_free_stream_boundaries.jl

```diff
@@ -0,0 +1,60 @@
+
+using Downloads: download
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations3D(1.4)
+
+initial_condition = initial_condition_constant
+
+polydeg = 3
+solver = DGSEM(polydeg = polydeg, surface_flux = flux_lax_friedrichs)
+
+###############################################################################
+# Get the uncurved mesh from a file (downloads the file if not available locally)
+
+default_mesh_file = joinpath(@__DIR__, "cube_boundaries.inp")
+isfile(default_mesh_file) ||
+    download("https://gist.github.com/DanielDoehring/eefe73ae5d113bc3944a518b6e88e663/raw/359a58a808790f3c3efc050273270eb1cc8ee353/cube_boundaries.inp",
```

Comment:
```suggestion
default_mesh_file = joinpath(@__DIR__, "mesh_cube_boundaries.inp")
isfile(default_mesh_file) ||
    download("https://gist.github.com/DanielDoehring/eefe73ae5d113bc3944a518b6e88e663/raw/359a58a808790f3c3efc050273270eb1cc8ee353/mesh_cube_boundaries.inp",
```
Similar comment to the 2D elixir